### ISSUE TITLE
add hplayground to stack

### ISSUE THREAD

**Issue by agreif** [agocorona/hplayground#10]
Hi,
I cannot find it currently in stack:
https://www.stackage.org/package/hplayground

any plans to put it there? 

thanks,
Alex.

**Comment by agocorona** [comment]
Yes. I'm integrating it with transient and putting it in stack. Currently
it is in an alpha state under GHCJS. look at the package :


https://github.com/agocorona/ghcjs-hplay

After that we will do the Haste integration

2016-02-28 13:26 GMT+01:00 Alex Greif <notifications@github.com>:

> Hi,
> I cannot find it currently in stack:
> https://www.stackage.org/package/hplayground
>
> any plans to put it there?
>
> thanks,
> Alex.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/agocorona/hplayground/issues/10>.
>



-- 
Alberto.

**Comment by agreif** [comment]
Hi, thanks for the quick answer.
I saw that even haste is not in stack, so maybe this will be a blocker.

Alex.
